storybookjs/aem
# CLI Init Command

The project installer and config generator portion of issue https://github.com/storybookjs/aem/issues/52.

- [x] Root path questions
- [x] Local project path questions
- [x] Story config questions
- [x] AEM package config questions
- [x] Creating .storybook files
- [x] Have the user navigate to the page component, template, and heading component.
- [x] Add the ability to navigate back up a directory.
- [x] Initial stories questions.
- [x] Test with the AEM Project Archetype with both the stories being next to the components and in a separate stories folder.

## What I did

Created a `sba init` command based on the prior version in the old repo.

## How to test

Run `sba init` or `sba init --use-defaults` in various project setups or in an empty directory.

After merging this PR we should create Issues for the following:

1. Update the `story` command to generate stories that use the `htl-engine` story format and ask the appropriate questions during the `init` command.
2. Update the `init` method to include client libs if possible.
